### ISSUE TITLE
Question: please explain usability indication on the revoked key

### ISSUE THREAD

**Issue by IvanPizhenko** [vanitasvitae/pgpainless#306]
I've generated a test key using GPG.
Here are files:
[prv.txt](https://github.com/pgpainless/pgpainless/files/9246062/prv.txt)
[pub.txt](https://github.com/pgpainless/pgpainless/files/9246063/pub.txt)

Then I have used following JavaScript code to read key, revoke it, and save revoked key:
```js
const fs = require('fs');
const openpgp = require('openpgp');

(async () => {
    const prvArmored = new String(fs.readFileSync('../keys/prv.txt'));
    const { publicKey: revokedPub, privateKey: revokedPrv } = await openpgp.revokeKey({
        key: await openpgp.readKey({ armoredKey: prvArmored }),
        format: 'armored' // output armored keys
    });
    fs.writeFileSync('../keys/revoked-pub.txt', revokedPub);
    fs.writeFileSync('../keys/revoked-prv.txt', revokedPrv);
    console.log('DONE')
})();
```

As result I've got the following revoked key files:
[revoked-prv.txt](https://github.com/pgpainless/pgpainless/files/9246068/revoked-prv.txt)
[revoked-pub.txt](https://github.com/pgpainless/pgpainless/files/9246071/revoked-pub.txt)

When I inspect them with following code:

```kotlin
package org.example

import org.junit.jupiter.api.Test
import org.pgpainless.PGPainless
import java.io.FileInputStream

class KeyInfoTest {
    private val keyDir = "${System.getenv("HOME")}/pro/fc/_tests/revocation_test/keys"

    @Test
    fun keyInfoTest() {
        checkKey("prv.txt")
        checkKey("revoked-prv.txt")
    }

    private fun checkKey(keyfile: String) {
        val key = FileInputStream("$keyDir/$keyfile").use { PGPainless.readKeyRing().secretKeyRing(it) }
        val keyInfo = PGPainless.inspectKeyRing(key)
        println("$keyfile: encryption: ${keyInfo.isUsableForEncryption} signing: ${keyInfo.isUsableForSigning} ")
    }
}
```
I'm getting the following output:
```
prv.txt: encryption: true signing: true 
revoked-prv.txt: encryption: true signing: false
```

@tomholub  says that he expects revoked key to be not usable for anything, but still it shows up as usable for encryption. Is this really correct? How does PGPainless determine that key is usable for encryption? Maybe there's a bug?

**Comment by vanitasvitae** [comment]
I really need to rework the KeyRingInfo class :D

Ideally, the API needs some `isRevoked()` method which can be used to check if a revocation signature made the key unusable.
Luckily. my work on the web-of-trust is kind of related to this. so I expect that some improvements will make its way into the core API as well :)

For now I think I fixed the issue by adding a check to see if the primary key is validly bound (ca09ac62caa1134ac84774aeb034341f6255ef86).

For PGPainless 2.0 though, I plan to rework the class from ground up.

**Comment by vanitasvitae** [comment]
Until I make a new release (might happen soon), you can work around this by additionally checking `info.isKeyValidlyBound(info.getKeyId());`.

**Comment by IvanPizhenko** [comment]
Great! Thank you @vanitasvitae , meanwhile I will apply this temporary workaround and I hope you publish the fixed version soon.

**Comment by vanitasvitae** [comment]
I just published version 1.3.4, which contains the fix :+1:

**Comment by IvanPizhenko** [comment]
Thank you!  I will use it.
